homes_guests_loves.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Title</title>
+    <link href="homes_guests_loves_styles.css" rel="stylesheet">
+</head>
+<body>
+<h1 class="col-lg-12 col-sm-8">Homes guests loves</h1>
+
+<!--<svg class="svg" width="296" height="296" viewBox="0 0 296 296" fill="none" xmlns="http://www.w3.org/2000/svg">-->
+
+    <div class="home_photos_box col-lg-10 col-sm-12">
+
+        <div class="home_place_box box1">
+        <img class="home_photo" src="images/homes_images/hotel_leopold.svg" alt="hotel_leopold">
+        <p class="home_hotel_name home_text">Hotel Leopold</p>
```

Comment:
why you name classes with an underscore? If you tried using BEM, please read the docs 1 more time. Just using underscores instead of dashes is not BEM.